.pipelines/pipeline.yaml

```diff
@@ -159,6 +159,20 @@ stages:
           name: "$(BUILD_POOL_NAME_LINUX_AMD64)"
         strategy:
           matrix:
+            cni_linux_amd64:
```

Comment:
Unless this gets renamed back to cni-dropgz, which from the PR description that seems like it will be, we need to add this to the cleanup script regex for acnpublic

Reply:
will follow up